Modules/Office365DSC/DSCResources/MSFT_TeamsTeam/MSFT_TeamsTeam.schema.mof

```diff
@@ -6,7 +6,7 @@ class MSFT_TeamsTeam : OMI_BaseResource
     [Write, Description("Description of Team.")] string Description;
     [Write, Description("Team group ID, only used to target a Team when duplicated display names occurs.")] string GroupID;
     [Write, Description("Owners of the Team")] string Owner[];
-    [Write, Description("Visibility of the Team"),ValueMap{"Public","Private"},Values{"Public","Private"}] String Visibility;
+    [Write, Description("Visibility of the Team"),ValueMap{"Public","Private","HiddenMembership"},Values{"Public","Private","HiddenMembership"}] String Visibility;
```

Comment:
you can also put this new value into the ps1 file where valuemaps are used, just in case

Reply:
done